extensions/default/src/getCustomizationModule.tsx

```diff
@@ -40,5 +40,76 @@ export default function getCustomizationModule() {
         ],
       },
     },
+
+    {
+      name: 'default',
+      value: [
+        /**
+         * Customization Component Type definition for overlay items.
+         * Overlay items are texts (or other components) that will be displayed
+         * on a Viewport Overlay, which contains the information panels on the
+         * four corners of a viewport.
+         *
+         * @definition of a overlay item using this type
+         * The value to be displayed is defined by
+         *  - setting DICOM image instance's property to this field,
+         *  - or defining contentF()
+         *
+         * {
+         *   id: string - unique id for the overlay item
+         *   customizationType: string - indicates customization type definition to this
+         *   label: string - Label, to be displayed for the item
+         *   title: string - Tooltip, for the item
+         *   color: string - Color of the text
+         *   condition: ({ instance }) => boolean - decides whether to display the overlay item or not
+         *   attribute: string - property name of the DICOM image instance
+         *   contentF: ({ instance, formatters }) => string | component,
+         * }
```

Comment:
I expected a position (topLeft, etc.) be here, how is the user then say where to put it